src/newSeed/stage3.ts

```diff
@@ -53,15 +54,20 @@ export class Stage3 extends BaseStage {
       message = "Please enter a non-zero number for Initial Circulating Seed Token Supply";
     }
     // Check the token distribution
-    this.seedConfig.tokenDetails.tokenDistrib.forEach((tokenDistrb: {category: string, amount: string, lockup: number}) => {
+    let totalDistribAmount = BigNumber.from("0");
+    this.seedConfig.tokenDetails.tokenDistrib.forEach((tokenDistrb: { category: string, amount: string, lockup: number }) => {
       if (!tokenDistrb.category) {
         message = "Please enter a value for Category";
       } else if (!tokenDistrb.amount || tokenDistrb.amount === "0") {
         message = `Please enter a non-zero number for Category ${tokenDistrb.category} Amount`;
       } else if (!(tokenDistrb.lockup > 0)) {
         message = `Please enter a non-zero number for Category ${tokenDistrb.category} Lock-up`;
       }
+      totalDistribAmount = totalDistribAmount.add(BigNumber.from(tokenDistrb.amount));
```

Comment:
probably can be:  `totalDistribAmount = totalDistribAmount.add(tokenDistrb.amount);`